packages/core-js/utils/rpc.js

```diff
@@ -12,11 +12,18 @@ async function restoreSnapshot(snapshotId, provider) {
   await mineBlock(provider);
 }
 
+async function timeWarp(ellapsedTime, provider) {
```

Comment:
For consistency with what people are used to, could we rename this to `fastForward(seconds, provider)`?